dev_guide/quota.adoc

```diff
@@ -175,10 +190,13 @@ resource-quota.json
 <1> The name of this quota document
 <2> The total amount of memory requested across all containers may not exceed 1Gi.
 <3> The total number of cpu requested across all containers may not exceed 20 Kubernetes compute units.
-<4> The total number of pods in the project
-<5> The total number of services in the project
-<6> The total number of replication controllers in the project
-<7> The total number of resource quota documents in the project
+<4> The size of image uploaded to internal registry may not exceed 5Gi.
+<5> The total size of images in internal registry under one repository (aka image stream) may not exceed 15Gi.
+<6> The total size of images in internal registry within one project may not exceed 40Gi.
```

Comment:
Same remarks as above, make it explicit that first is single image size, the latter IS image size and the last project images.